core/templates/dev/head/components/exploration_summary_tile_directive.html

```diff
@@ -175,14 +175,42 @@ <h2 class="exploration-title protractor-test-exp-summary-tile-title"><[getExplor
   .oppia-exploration-summary-tile .avatars-num-minus-one {
     color: white;
     font-family: Roboto;
-    font-size: x-small;
+    font-size: xx-small;
```

Comment:
@seanlip , Okay, I'll test it and commit the changes.
On 7 Apr 2016 15:00, "Sean Lip" <notifications@github.com> wrote:

> In
> core/templates/dev/head/components/exploration_summary_tile_directive.html
> <https://github.com/oppia/oppia/pull/1693#discussion_r58843607>:
>
> > @@ -175,14 +175,42 @@ <h2 class="exploration-title protractor-test-exp-summary-tile-title"><[getExplor
> >    .oppia-exploration-summary-tile .avatars-num-minus-one {
> >      color: white;
> >      font-family: Roboto;
> > -    font-size: x-small;
> > +    font-size: xx-small;
>
> Change this to 0.55em. I think that should do it.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/oppia/oppia/pull/1693/files/23bc8841963c549ae0e68a07c1eb1d6ad4d89fc6#r58843607>
>